bitnami/kafka/templates/_helpers.tpl

```diff
@@ -316,6 +316,13 @@ Return true if a configmap object should be created
 {{- end -}}
 {{- end -}}
 
+{{/*
+Return the Kafka log4j configmap
+*/}}
+{{- define "kafka.log4j.configmapName" -}}
+    {{- printf "%s-log4j-configuration" (include "kafka.fullname" .) -}}
+{{- end -}}
```

Comment:
We should consider the possibility of reusing a existing configmap including `log4j.properties`:

```
{{/*
Return the Kafka log4j configmap
*/}}
{{- define "kafka.configmapName" -}}
{{- if .Values.existingConfigmap -}}
    {{- printf "%s" (tpl .Values.existingLog4jConfigmap $) -}}
{{- else -}}
    {{- printf "%s-log4j-configuration" (include "kafka.fullname" .) -}}
{{- end -}}
{{- end -}}
```